src/web/js/dashboard/config.js

```diff
@@ -1,9 +0,0 @@
-// App name in *Title Case*.
```

Comment:
This file seemed useful to me; again not sure if I'm just looking at a backwards diff, since the parameterization seems nice.